docs/src/style.css

```
@@ -1,30 +1,28 @@
-  top: auto !important;
-  left: auto !important;
-  z-index: 20 !important;
-  min-height: 0 !important;
-  padding: 0.625rem 1rem !important;
-  background: rgba(0, 184, 92, 0.08) !important;
-  border-top: 1px solid rgba(0, 184, 92, 0.18);
-  border-bottom: 1px solid rgba(0, 184, 92, 0.2);
-  color: rgb(210, 246, 226) !important;
+  --banner-bg-light: #e8f6ee;
+  --banner-bg-dark: #163528;
+  height: 2.5rem !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)